### PR TITLE
DOC-5987: Added missing parameter and updated some default Corda config fields

### PR DESCRIPTION
**Pull request by AnnaK-R3**
Preview: https://anna.preview.docs.r3.com/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.html

Added the `brokerConnectionTtlCheckIntervalMs` parameter to the page.
Updated defaults for `journalBufferTimeout` and `batchTimeoutMs`.

### CODE REVIEW

File: content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md [hunk 1]
```diff
@@ -604,7 +612,7 @@ Once a notary is configured with a default value, it cannot be reconfigured. To
     * *Default:* 10 000
   * `batchTimeoutMs`
     * Configures the amount of time that the notary will wait before processing a batch, even if the batch is not full. Smaller values can lead to lower latency but potentially worse throughput as smaller batches might be processed.
-    * *Default:* 200
+    * *Default:* 1
```
Comment (rick-r3) [inline content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md:615]: This one remains the same.

File: content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md [hunk 2]
```diff
@@ -681,7 +689,7 @@ Once a notary is configured with a default value, it cannot be reconfigured. To
       * *Default:* 10
     * `batchTimeoutMs`
       * Configures the amount of time that the notary will wait before processing a batch, even if the batch is not full. Smaller values can lead to lower latency but potentially worse throughput as smaller batches might be processed.
-      * *Default:* 200
+      * *Default:* 1
```
Comment (rick-r3) [inline content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md:692]: This one remains the same.

**Review by rick-r3 — CHANGES_REQUESTED** [review]
Only one of the `batchTimeOutMs` needs updating.  I've commented on the ones that should be reverted.

Rest of it looks good.

File: content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md (hunk 1 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md]: updated

File: content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md]: updated

**Review by rick-r3 — APPROVED** [review]

File: content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md [hunk 3]
```diff
@@ -115,10 +115,10 @@ Optional name of the CryptoService implementation. This only needs to be set if
 Optional path to the configuration file for the CryptoService provider. This may have to be present if you use a different CryptoService provider than the default one.
 
 ## `cryptoServiceFlowRetryCount`
- 
-`cryptoServiceFlowRetryCount` is used to specify which actions should be taken in the event of a flow suffering a CryptoServiceException due to a timeout. 
 
-The *absolute value* of *cryptoServiceFlowRetryCount* determines the number of times that the flow is retried. The *sign* of the value determines what happens when all retries are exhausted: 
+`cryptoServiceFlowRetryCount` is used to specify which actions should be taken in the event of a flow suffering a CryptoServiceException due to a timeout.
+
+The *absolute value* of *cryptoServiceFlowRetryCount* determines the number of times that the flow is retried. The *sign* of the value determines what happens when all retries are exhausted:
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.11/enterprise/node/setup/corda-configuration-fields.md:121]:
```suggestion
The *absolute value* of `cryptoServiceFlowRetryCount` determines the number of times that the flow is retried. The *sign* of the value determines what happens when all retries are exhausted:
```

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

**Review by paulmoloneyr3 — APPROVED** [review]

**Pull request merged**